lib/ziti-tunnel-cbs/ziti_tunnel_ctrl.c

```diff
@@ -603,10 +603,7 @@ static int load_identity(const char *identifier, const char *path, int api_page_
         inst->opts.api_page_size = api_page_size;
     }
 
-    uv_async_t *ar = calloc(1, sizeof(uv_async_t));
-    ar->data = inst;
-    uv_async_init(CMD_CTX.loop, ar, load_ziti_async);
-    uv_async_send(ar);
+    load_ziti_async(CMD_CTX.loop, inst);
```

Comment:
maybe this should be renamed to `load_ziti` since it is being called inline vs through uv_async_send?